packages/contractkit/src/wrappers/BaseWrapper.ts

```diff
@@ -22,30 +25,39 @@ export abstract class BaseWrapper<T extends Contract> {
   }
 }
 
-/** Parse string -> BigNumber */
-export function toBigNumber(input: string) {
-  return new BigNumber(input)
-}
+export const numberLikeToBigNumber = (input: NumberLike) => new BigNumber(input)
```

Comment:
Nit: there's no need for us to define a NumberLike type, `BigNumber.Value` is exactly this